docs/api/transaction/get-mempool-transactions.example.json

```diff
@@ -0,0 +1,391 @@
+{
+  "limit": 96,
+  "offset": 0,
+  "total": 24,
+  "results": [
+    {
```

Comment:
It might be enough to have 1-2 JSON objects in here as a sample

Reply:
cut it to 5, which I think covers enough variants